### PR TITLE
Recipe Revision

### LINKED ISSUE #13: Extended Angel's recipe revisions
In the most recent update to Angel's Bioprocessing, the tiering for the algae farm shifted, such that the T3 Algae farm now has the same tier requirements as the T4 algae farm in Extended angels:

https://github.com/Arch666Angel/mods/commit/ead07bec2f029ed89cd07e54972aa8cc1de73210#diff-1a9182b2da56317f04d372cc43ccda5afe17d296f3278dc99651aa647875bf0b

image
image

Should the Algae Farm 4 be moved to a higher tier/balanced around the new requirements?

### PR DESCRIPTION
**Pull request by Curry**
Fixes #13

### CODE REVIEW

File: prototypes/recipe-builder-fallbacks.lua [hunk 1]
```diff
@@ -0,0 +1,3 @@
+angelsmods.functions.RB.set_fallback("item", "t6-brick", { { "titanium-concrete-brick", 5 }, { "stone-brick", 5 } })
```
Comment (Curry) [inline prototypes/recipe-builder-fallbacks.lua:1]: https://github.com/Arch666Angel/mods/blob/d6f95b4964c0c995e4a4847bf3f0bd28686d8b93/angelsrefining/prototypes/recipe-builder-fallbacks.lua#L24

File: prototypes/recipe-builder-fallbacks.lua [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+angelsmods.functions.RB.set_fallback("item", "t6-plate", { { "copper-tungsten-alloy", 4 },  { "steel-plate", 13 } })
+angelsmods.functions.RB.set_fallback("item", "t6-pipe", { { "copper-tungsten-pipe", 3 }, { "pipe", 8 } })
```
Comment (Curry) [inline prototypes/recipe-builder-fallbacks.lua:3]: https://github.com/Arch666Angel/mods/blob/d6f95b4964c0c995e4a4847bf3f0bd28686d8b93/angelsrefining/prototypes/recipe-builder-fallbacks.lua#L40

File: prototypes/recipe-builder-fallbacks.lua [hunk 3]
```diff
@@ -0,0 +1,3 @@
+angelsmods.functions.RB.set_fallback("item", "t6-brick", { { "titanium-concrete-brick", 5 }, { "stone-brick", 5 } })
+angelsmods.functions.RB.set_fallback("item", "t6-plate", { { "copper-tungsten-alloy", 4 },  { "steel-plate", 13 } })
```
Comment (Curry) [inline prototypes/recipe-builder-fallbacks.lua:2]: https://github.com/Arch666Angel/mods/blob/d6f95b4964c0c995e4a4847bf3f0bd28686d8b93/angelsrefining/prototypes/recipe-builder-fallbacks.lua#L16

File: prototypes/recipes/refining.lua [hunk 4]
```diff
@@ -192,10 +192,10 @@ angelsmods.functions.RB.build({
...
             ingredients = {
                 {"ore-leaching-plant-3", 1},
-                {"copper-tungsten-alloy", 8},
-                {"t5-circuit", 16},
-                {"nitinol-bearing", 8},
-                {"titanium-concrete-brick", 40},
+                {"t6-plate", 4},
+                {"t5-circuit", 8},
+                {"t6-pipe", 4},
+                {"t6-brick", 8},
```
Comment (Curry) [inline prototypes/recipes/refining.lua:198]: https://github.com/Arch666Angel/mods/blob/ead07bec2f029ed89cd07e54972aa8cc1de73210/angelsrefining/prototypes/recipes/refining-entity-angels.lua#L203
Bringing inline with angel's refining recipe

File: prototypes/recipes/refining.lua [hunk 5]
```diff
@@ -236,10 +235,9 @@ angelsmods.functions.RB.build({
             enabled = "false",
             ingredients = {
                 {"ore-refinery-2", 1},
-                {"copper-tungsten-alloy", 12 * buildingmulti},
-                {"nitinol-bearing", 12 * buildingmulti},
-                {"t5-circuit", 24 * buildingmulti},
-                {"titanium-concrete-brick", 100 * buildingmulti},
+                {"t6-plate", 12 * buildingmulti},
+                {"t5-circuit", 12 * buildingmulti},
+                {"t6-brick", 20 * buildingmulti},
```
Comment (Curry) [inline prototypes/recipes/refining.lua:240]: https://github.com/Arch666Angel/mods/blob/ead07bec2f029ed89cd07e54972aa8cc1de73210/angelsrefining/prototypes/recipes/refining-entity-angels.lua#L314
Bringing inline with angel's refining recipe

File: prototypes/recipes/bioprocessing.lua [hunk 6]
```diff
@@ -972,10 +972,10 @@ angelsmods.functions.RB.build({
         enabled = false,
         ingredients = {
             {"bio-refugium-puffer", 1},
-            {"t2-plate", 4},
-            {"t2-circuit", 4},
-            {"t2-brick", 3},
-            {"t2-pipe", 11}
+            {"t4-plate", 4},
```
Comment (Curry) [inline prototypes/recipes/bioprocessing.lua:975]: Will need progression tiering updated

File: prototypes/recipes/bioprocessing.lua [hunk 7]
```diff
@@ -1032,10 +1032,10 @@ angelsmods.functions.RB.build({
         enabled = false,
         ingredients = {
             {"bio-refugium-biter", 1},
-            {"t2-plate", 10},
-            {"t2-circuit", 4},
-            {"t2-brick", 19},
-            {"t2-pipe", 11}
+            {"t5-plate", 10},
+            {"t5-circuit", 4},
```
Comment (Curry) [inline prototypes/recipes/bioprocessing.lua:1036]: Will need progression tiering updated

**Comment by kirazy** [comment]
We're getting close to when Angel's is going to update, so I'm going to bring this in. If you could give the recipes another pass that'd be appreciated.

Angel's Bioprocessing added a t4 farm so EA is going to not add a t4 farm, unless you want to make a t5 farm?...

**Pull request merged**